src/infiniteGrid.js

```diff
@@ -317,6 +317,9 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 			if (prevScrollTop < scrollTop) {
 				if ($.isEmptyObject(this._bottomElement)) {
 					this._bottomElement = this.getBottomElement();
+					if (this._bottomElement == null) {
```

Comment:
if `_buttomElement` should be checked strictly as `null` use === instead ==.
undefined can be treated true in below comparison.
```js
undefined == null;
```